XHR.cgi

```diff
@@ -0,0 +1,17 @@
+#!/usr/local/bin/perl
+# A caller for loading XHR related routines
+use strict;
+
+our ($trust_unknown_referers, $root_directory);
+
+$trust_unknown_referers = 1;
```

Comment:
Is it safe to turn off referer checks?

Reply:
> Is it safe to turn off referer checks?

Yes, considering what this script does. However, it is not really necessary. I did it for testing purposes. Fix in this commit https://github.com/webmin/webmin/pull/1917/commits/4ee13f0e2059ee0b33fd171961d69b26cbdcbe1a.